bundles/org.openhab.binding.freeboxos/src/main/resources/OH-INF/thing/channel-types.xml

```diff
@@ -0,0 +1,355 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<thing:thing-descriptions bindingId="freeboxos"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<channel-type id="lcd-brightness" advanced="true">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Screen Brightness</label>
+		<description>Brightness level of the screen in percent</description>
+		<category>DimmableLight</category>
+		<state pattern="%d %unit%"/>
+	</channel-type>
+
+	<channel-type id="lcd-orientation">
+		<item-type>Number</item-type>
+		<label>Screen Orientation</label>
+		<description>Screen Orientation in degrees</description>
+		<state pattern="%d °">
+			<options>
+				<option value="0">Horizontal</option>
+				<option value="90">Turned left</option>
+				<option value="180">Reversed</option>
+				<option value="270">Turned right</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="rssi">
+		<item-type>Number:Power</item-type>
+		<label>RSSI</label>
+		<description>Received signal strength indicator</description>
+		<category>QualityOfService</category>
+		<state readOnly="true" pattern="%d %unit%"/>
+	</channel-type>
+
+	<channel-type id="ssid" advanced="true">
+		<item-type>String</item-type>
+		<label>SSID</label>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="wifi-host" advanced="true">
+		<item-type>String</item-type>
+		<label>Access Point</label>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="wifi-quality">
+		<item-type>Number</item-type>
```

Comment:
See above comments on RSSI channel.

Reply:
You're right, this one could be replaced by system.signal-strength.
Done